plugins/database/hana/hana_test.go

```diff
@@ -0,0 +1,172 @@
+package hana
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/vault/builtin/logical/database/dbplugin"
+	"github.com/hashicorp/vault/plugins/helper/database/connutil"
+)
+
+var (
+	testHANAImagePull sync.Once
+)
+
+func TestHANA_Initialize(t *testing.T) {
```

Comment:
Is it possible to test this using a HANA docker image, if one is available? Otherwise the test environment would somehow need to have a HANA DB instance running, with the proper environment variables set as well.